pkg/plugin/step_test.go

```diff
@@ -342,7 +372,7 @@ func Test_replaceTemplateVariable(t *testing.T) {
 				interval:     0,
 				timeInterval: "",
 			},
-			want: "rate(ingress_nginx_request_qps{}[1m0s])",
+			want: "rate(ingress_nginx_request_qps{}[0s])",
```

Comment:
0s never makes sense. Should we check for it and replace with some default value?

Reply:
fixed